recipes/realgud-lldb

```diff
@@ -0,0 +1,5 @@
+(realgud-lldb
+ :fetcher github
+ :repo "realgud/realgud-lldb"
+ :files ("realgud-lldb.el"
```

Comment:
I'll just edit this to use `:defaults` instead of this specific filename, and then merge.